### PR TITLE
Integrate Jupyterviz

### PR DESCRIPTION
**Pull request by tpike3**
Unfortunately with my current job, I am always behind so this is not very clean or optimized however this integrates jupyter_viz into mesa_geo

Please also see [mesa-examples pull request for browser and jupyter examples](https://github.com/projectmesa/mesa-examples/pull/117)

TODO: fix the jupyter version when you try and expand from the cell output to full screen

### CODE REVIEW

**Comment by rht** [comment]
The code shouldn't be duplicated as much as possible, otherwise it is much more difficult to maintain.

**Comment by tpike3** [comment]
> The code shouldn't be duplicated as much as possible, otherwise it is much more difficult to maintain.

Removed the `draw_grid` code from components as the assumption is it will not be used since users will put everything on maps.

File: mesa_geo/geoexperimental/UserParam.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+class UserParam:
```
Comment (rht) [inline mesa_geo/geoexperimental/UserParam.py:1]: This entire file can be removed and to use the `Slider`, you can import from `mesa.experimental`.

File: mesa_geo/geoexperimental/jupyter_viz.py [hunk 2]
```diff
@@ -0,0 +1,410 @@
+import sys
```
Comment (rht) [inline mesa_geo/geoexperimental/jupyter_viz.py:1]: Forking the entire file here is not sustainable. Needs to do it the way it is done here https://github.com/projectmesa/mesa-examples/blob/1933cbc8251ca047273e0f88193f692ccbb86de6/examples/sugarscape_g1mt/app.py#L9.

File: mesa_geo/geoexperimental/UserParam.py (hunk 1 above)
Comment (tpike3) [reply mesa_geo/geoexperimental/UserParam.py:1]: Yep got it.. in retrospect it seem obvious, ahh well such is life

**Comment by tpike3** [comment]
Still rushing through this but let me know what else I need to do so we can get this release before AAG. 

Sorry for the hassle

**Pull request closed without merging**